charts/rancher-cis-benchmark-crd/6.1.0-rc1/templates/clusterscan.yaml

```
@@ -0,0 +1,148 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterscans.cis.cattle.io
+spec:
+  group: cis.cattle.io
+  names:
+    kind: ClusterScan
+    plural: clusterscans
+  scope: Cluster
```

Let's align the contents of this file with https://github.com/rancher/cis-operator/blob/master/crds/clusterscan.yaml. 

The main difference is this:

```suggestion
    plural: clusterscans
    singular: clusterscan
  scope: Cluster
```

Plus a few elements that appear in different positions. Let's just copy and paste it for the v2.9 branch.

---

done